_pages/software.md

```diff
@@ -0,0 +1,140 @@
+  image: /assets/images/software/software.jpg
+  caption: "Photo credit: [**Markus Spiske**](https://unsplash.com/@markusspiske?utm_source=unsplash&utm_medium=referral&utm_content=creditCopyText) on [**Unsplash**](http://unsplash.com/)"
+---
+
+The KBSS team produces two types of outputs: ontologies and software to maintain them.
+
+## Ontologies
+
+Ontology design is one of our primary research topics. We have built several ontologies as part of our research projects. The most iportant ontologies are here.
+
+### TermIt ontology
+
+Ontology to describe resources as they are used in [TermIt](#termit) software. It reuses [Dataset description ontology](#dataset-description-ontology).
+
+*TODO: add ontology schema*
+
+[GitHub repository with Turtle serialization](https://github.com/kbss-cvut/termit/tree/master/ontology){: .btn .btn--info}
+
+
+### Aviation Safety Ontology
+
+Modular ontology for safety reporting in high-risk industries, mainly for aviation.
+
+[Docs and serialization](https://www.inbas.cz/aviation-safety-ontology){: .btn .btn--info}
+
+
+### Urban Ontology
+
+The Urban ontology unifies terminology in the urban planning domain, developed in cooperation with the [Prague Institute of Planning and Development](https://iprpraha.cz/en/).
+
+[OWLGred visualization](http://owlgred.lumii.lv/online_visualization/l4wh){: .btn .btn--info}
+[OWL serialization](http://kbss.felk.cvut.cz/ontologies/town-plan/ontology.owl){: .btn .btn--info}
+
+
+### Dataset Description Ontology
+
+It is an ontology for describing dataset exploration scenario. The ontology design is based on UFO and is done with respect to OntoUML methodology.
+
+The ontology is actively used in the [Dataset dashboard](http://onto.fel.cvut.cz/dataset-dashboard) system for exploring linked datasets.
+
+[OWL serialization](http://onto.fel.cvut.cz/ontologies/ddo){: .btn .btn--info}
+[Dataset dashboard](http://onto.fel.cvut.cz/dataset-dashboard){: .btn .btn--info}
+
+
+### [Operation Unified Foundational Ontology (UFO)](http://onto.fel.cvut.cz/ontologies/ufo)
+
+Operational version of the UFO Ontology describes basic notions of UFO in terms of the Web Ontology Language, possibly extended with SWRL rules.
+The main goals of the ontology are
+
+1. to provide easy way to link UFO-related terms to applications, resources, as well as multimedia,
+2. to provide a computable artifact for basic reasoning and UFO-compliant data integration.
+
+[OWL serialization](http://onto.fel.cvut.cz/ontologies/ufo){: .btn .btn--info}
+
+
+## Software
+
+We develop end user systems and tools as well software libraries used by other systems.
+
+### End User Tools and Systems
+
+#### TermIt
+
+*TODO: add some screenshots*
+TermIt is a SKOS-compatible terminology manager and editor.
+
+[TermIt Website](https://kbss-cvut.github.io/termit-web/){: .btn .btn--info}
+[Source code backend](https://github.com/kbss-cvut/termit){: .btn .btn--info}
+[frontend](https://github.com/kbss-cvut/termit-ui){: .btn .btn--info}
```

Comment:
Source code - frontend (and probably similarly Source code - backend)?

Reply:
IC, I was thinking about that too